wordclock_config/wordclock_config.example.cfg

```diff
@@ -22,15 +22,20 @@ developer_mode = False
 language = german
 
 # Choose wiring layout here. Options are: bernds_wiring, christians_wiring, ... (possibly you need your own wiring layout)
-wiring_layout = bernds_wiring
+wiring_layout = davids_wiring
```

Comment:
Please reset to default value (bernds_wiring is part of the docs on how to build the hardware and therefore widely used).